src/main/java/com/google/devtools/build/lib/rules/python/PythonUtils.java

```diff
@@ -37,21 +39,28 @@ public final class PythonUtils {
    */
   public static class GetInitPyFiles implements Runfiles.EmptyFilesSupplier {
     private final Predicate<PathFragment> isPackageInit;
+    private final UUID guid;
```

Comment:
nit: mind adding a comment as to why the UUID is necessary here? It's a little surprising if one doesn't intimately know how Runfiles works.

Reply:
I added a comment to the constructor argument. Should I also add it to the field?

Reply:
Meh, I won't make you jump through that particular hoop; it's not that big a deal.